src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+                    <td>Odometer (0.1 Km/bit)</td>
+                </tr>
+                <tr>
+                    <td>19</td>
+                    <td>
+                        Status Flags (from LSB):
+                        <br></br>
+                        1st bit: 1 = GPS Valid
+                        <br></br>
+                        2nd bit: 1 = Ignition On
+                        <br></br>
+                        3rd bit: 1 = Engine Bus Activity
+                        <br></br>
+                        4th bit: 1 = Date/Time Valid
+                        <br></br>
+                        5th bit: 1 = Speed From Engine
+                        <br></br>
+                        6th bit: 1 = Odometer From Engine
+                    </td>
+                </tr>
+                <tr>
+                    <td>20-23</td>
+                    <td>Trip Odometer (0.1 Km/bit)</td>
+                </tr>
+                <tr>
+                    <td>24-27</td>
+                    <td>Total Engine Hours (0.1 hours/bit)</td>
+                </tr>
+                <tr>
+                    <td>28-31</td>
+                    <td>Trip Duration (1 second/bit)</td>
+                </tr>
+                <tr>
+                    <td>32-35</td>
+                    <td>Deprecated</td>
+                </tr>
+                <tr>
+                    <td>36-39</td>
+                    <td>Driver ID</td>
+                </tr>
+                <tr>
+                    <td>40-51</td>
+                    <td>GO Device Serial Number</td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Type 3 VIN</h4>
+        <p>Sent in response to an IOX Request(0x25) message with Type Request VIN (0x04).</p>
+        <table>
+            <thead>
+                <tr>
+                    <th>Bytes</th>
+                    <th>GO Device Data</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-16</td>
+                    <td>VIN</td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Type 12 GO Info</h4>
+        <p>Supported from protocol version 1.1. Sent in response to IOX Request/Status (0x25) - Type 12.</p>
+        <h5>Payload Id = 0</h5>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0</td>
+                    <td>=0 for payload is GO serial number.</td>
+                </tr>
+                <tr>
+                    <td>1-12</td>
+                    <td>GO serial number</td>
+                </tr>
+                <tr>
+                    <td>13</td>
+                    <td>0</td>
+                </tr>
+            </tbody>
+        </table>
+        <h5>Payload Id = 1</h5>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on